common/config/azure-pipelines/build-test-publish.yml

```diff
@@ -35,6 +32,21 @@ jobs:
       parameters:
         runRushAudit: ${{ parameters.runRushAudit }}
 
+  - job: BuildAndTest mac-node18
+    pool:
+      name: iModelTechMacArm
```

Comment:
Who suggested to use this pool? I wonder if it's not deprecated because iModelTechCI is deprecated.

Reply:
I had spoke to @robertas-kerpys about this change; based of https://learn.microsoft.com/en-us/azure/devops/release-notes/roadmap/macos-agents-apple-silicon, there is no official support for macos arm on azure-pipelines so we need to host our own